site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -225,7 +225,13 @@ The following tables lists the configurable parameters of the Agones chart and t
 
 | Parameter                             | Description                                                                                                                                                                                                                           | Default |
 |---------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|
-|                                                          |                                                                                                                                                                                                                                                            |   {{% /feature %}}
+|                                                          |                                                                                                                                                                                                                                                            |
```

Comment:
I'm a little confused - everything in lines 114-142-ish in `agones.controller` is also configurable, shouldn't those be here as well?

Reply:
Added the `extensions` version of the configurations as well.